Misc/NEWS.d/next/C API/2018-06-30-21-48-16.bpo-34008.2Wjtm0.rst

```diff
@@ -0,0 +1,2 @@
+Added ``Py_IsInitialized`` to the list of APIs that are safe to call before
+the interpreter is initialized, and updated the embedding tests to cover it.
```

Comment:
I didn't want to put this in the `Documentation` section, since the tests did change, and one entry in the C API sections seems neater than mentioning it as both a docs and test suite change.